test/functional/bundleadd/add-basics.bats

```diff
@@ -44,15 +44,24 @@ global_teardown() {
 	assert_status_is 0
 	expected_output=$(cat <<-EOM
 		Starting download of remaining update content. This may take a while...
-		.
+		.*...33%
+		.*...66%
+		.*...100%
+
 		Finishing download of update content...
-		Installing bundle(s) files...
-		.
+		Installing bundle\(s\) files...
+		.*...16%
+		.*...33%
+		.*...50%
+		.*...66%
+		.*...83%
+		.*...100%
+
 		Calling post-update helper scripts.
 		Successfully installed 1 bundle
 	EOM
 	)
-	assert_is_output --identical "$expected_output"
+	assert_regex_is_output --identical "$expected_output"
```

Comment:
Can't we use assert_regex_in_output() here so we don't need to have all % printed in the test file?

Reply:
actually the percentages are normally being filtered out from the output by the test library (look at the `use_ignore_list()` function). But I specifically instructed testlib NOT to strip the percentages out from the output in this case by using the `--identical` flag. I thought it was valuable to have one test that would validate the percentages are being printed out correctly in case a change in the code breaks this output for some reason, otherwise we wouldn't catch that cause we are always filtering that out.